rook/helmfile.d/values/operator.yaml.gotmpl

```diff
@@ -37,3 +37,8 @@ csi:
     {{- end }}
 
   pluginTolerations: {{- toYaml .Values.plugin.tolerations | nindent 4 }}
+
+  allowUnsupportedVersion: {{ .Values.cluster.cephCsi.allowUnsupportedVersion }}
+  cephcsi:
+    # -- Ceph CSI image tag
+    tag: {{ .Values.cluster.cephCsi.tag }}
```

Comment:
Qusetion: Do we want to add some default value to the template values file for these maybe?

Reply:
I added them to the templates as empty variables so they are only templated if set https://github.com/elastisys/compliantkubernetes-kubespray/pull/402/commits/d7edb0863be003d366826c60e1beb93a4c7e4d77
Also added some comments for why you would want to set it, at least the reason for why we had to set this